Sources/AppcuesKit/AppcuesKit.docc/AppcuesFrameViewConfiguring.md

```diff
@@ -0,0 +1,37 @@
+# Configuring AppcuesFrameViews
```

Comment:
"AppcuesFrameViews" is a bit awkward, so maybe "Configuring an AppcuesFrameView" or "Configuring frame views" ([like Apple's docs](https://developer.apple.com/documentation/uikit/views_and_controls/collection_views))?